.cargo/config.toml

```diff
@@ -0,0 +1,2 @@
+[build]
+rustflags = ["--cfg", "tokio_unstable"]
```

Comment:
I hope this means "API unstable", not "unreliable"?

Reply:
That's what it means (API unstable). A bunch of it is already stable but not all of it.